src/test/scala/io/iohk/ethereum/transactions/PendingTransactionsManagerSpec.scala

```diff
@@ -115,14 +115,17 @@ class PendingTransactionsManagerSpec extends FlatSpec with Matchers with ScalaFu
     pendingTransactionsManager ! AddOrOverrideTransaction(firstTx)
     peerManager.expectMsg(PeerManagerActor.GetPeers)
     peerManager.reply(Peers(Map(peer1 -> Handshaked)))
+    Thread.sleep(Timeouts.normalTimeout.toMillis)
```

Comment:
Well, I tried [this](https://github.com/input-output-hk/etc-client/commit/51c0861f63add7fe33c7a41ffecb60b087bb5849) and it [passed](https://circleci.com/gh/input-output-hk/etc-client/2299) on Circle